src/plugins/hetero/executable_network.cpp

```diff
@@ -113,7 +120,7 @@ HeteroExecutableNetwork::HeteroExecutableNetwork(const InferenceEngine::CNNNetwo
     };
 
     // Set results, constants and parameters affinity
-    for (auto&& node : clonedFunction->get_ops()) {
+    for (auto&& node : clonned_function->get_ops()) {
```

Comment:
nitpick: we can reuse orderedOps.
```suggestion
    for (auto&& node : orderedOps) {
```